packages/storybook8/stories/Composites/CallWithChatComposite/Docs.mdx

```diff
@@ -0,0 +1,195 @@
+import { Image, Stack, MessageBar } from '@fluentui/react';
+import { Meta, Source } from '@storybook/addon-docs';
+import { useEffect, useRef } from 'react';
+import { SingleLineBetaBanner } from '../../BetaBanners/SingleLineBetaBanner';
+import containerText from '!!raw-loader!./snippets/CallWithChat.snippet.tsx';
+import serverText from '!!raw-loader!./snippets/Server.snippet.tsx';
+import { overviewPageImagesStackStyle } from '../../constants';
+
+<Meta title="Composites/CallWithChatComposite" />
+
+# CallWithChatComposite
+
+CallWithChatComposite brings together key components to provide a full call with chat experience out of the box.
+Inside the experience users can configure their devices, participate in the call with video and see other
+participants, including those with video turned on. The experience includes chat capabilities like sending and
+receiving messages, and notifications for message thread events such as typing indicators, message receipts,
+participants entering and leaving the chat.
+
+<Stack horizontalAlign="center" style={overviewPageImagesStackStyle}>
+  <Image src="images/call-with-chat-composite-hero.png" alt="Call with chat composite preview image" width="85%" />
+</Stack>
+<br />
+## Basic usage A CallWithChatComposite is comprised of two key underlying parts: an ACS Call and an ACS Chat thread. As
```

Comment:
```suggestion
## Basic usage 
A CallWithChatComposite is comprised of two key underlying parts: an ACS Call and an ACS Chat thread. As
```

Reply:
thanks for checking on this. Ill fix this up

Reply:
updated with your suggestions.